src/main/kotlin/io/sip3/salto/ce/rtcp/RtcpHandler.kt

```diff
@@ -66,13 +68,61 @@ open class RtcpHandler : AbstractVerticle() {
         }
 
         vertx.eventBus().localConsumer<Packet>(RoutesCE.rtcp) { event ->
-            try {
-                val packet = event.body()
-                handleRaw(packet)
-            } catch (e: Exception) {
-                logger.error(e) { "RtcpHandler 'handleRaw()' failed." }
+            val packet = event.body()
+            if (packet.source == "hep3") {
+                try {
+                    handleHep(packet)
+                } catch (e: Exception) {
+                    logger.error(e) { "RtcpHandler 'handleHep()' failed." }
+                }
+            }
+            if (packet.source == "sip3") {
+                try {
+                    handleRaw(packet)
+                } catch (e: Exception) {
+                    logger.error(e) { "RtcpHandler 'handleRaw()' failed." }
+                }
+            }
+        }
+    }
+
+    open fun handleHep(packet: Packet) {
```

Comment:
Given that we will switch `handleRaw` and `handleHep` calls, let's switch both methods in class too